src/sage/modular/quasimodform/element.py

```diff
@@ -31,16 +31,20 @@
 
 class QuasiModularFormsElement(ModuleElement):
     r"""
-    A quasimodular forms ring element. Such an element is describbed by SageMath
-    as a polynomial
+    A quasimodular forms ring element. Such an element is describbed by
```

Comment:
```suggestion
    A quasimodular forms ring element. Such an element is described by
```
This should be the final change.

Reply:
done